### DISCUSSION TITLE
Windows 11 version 23H2

### DISCUSSION THREAD

**Discussion by vbirloudis** [AveYo/MediaCreationTool.bat#290] [Ideas]
Hello AveYo,
Can you please upgrade the script to support Windows 11 23H2 version?

Thanks in advance.

**Comment by TerryUK** [comment]
You can use the standard trick of the project that worked last year to load the ISO in as per last year. The script run on the previous years update if not re-run was enough or use this years script. 

Standard trick:

》》
Select the bypass11 folder on the repository and then the Skip_TPM_Check_on_Dynamic_Update.cmd there. Select the download icon to save the file to the local system. 

Once done, right-click on the downloaded file and select run as administrator. Windows 11 may throw a SmartScreen warning prompt. Select "more info" and then "run anyway" to run the script and make the necessary changes on the device.

Go back to the mounted Windows 11 ISO and run setup.exe. Make sure you disable the checking for updates on the first page of setup. The upgrade to the latest version of Windows 11 should now commence as if the system is meeting all hardware requirements.
《《

On one PC I had an issue since the PC was opening up the ISO  file as 7zip... somewhere within the year association got change. You need the ISO to open up via the file manager and it then resembles a DVD / USB.

The other PC was plain sailing. I just double clicked on the ISO on the basis that last years fix was still enabled. It's important to give yourself admin rights and toggle not to do updates as per those instructions above. It worked. 

Back to the original PC I had loaded this year's latest release of the project. Think I'd got a combination going. Then ran last years script to cancel the toggle. This year's script to enable. Then it ran fine. 

The script being "Skip_TPM_Check_on_Dynamic_Update.cmd". Each time you run it, it toggles on and off the TPM check which stays glued on from the last run. They are Microsoft developer toggles which say ignore the TPM check at ISO installation. They sit in the machines environment from last time you ran it. 

Moral here somewhere. If you used it last year the toggle is likes to be set still and it will work just running the ISO.... but the ISO must open in the File Manager. If not correct that (my 7 zip issue, uninstalled 7 zip and renewed the correct association). I believe if you run the toggle script with this years release of the project you have to de-toggle last years script, then toggle this years script. That was the issue I had. Fools rush in etc.

Clear as mud.

On your question I believe the main project pulls from a library of ISOs and seemed to be upgrading from 21H2. It's that which isn't updated but you can use the skip TPM check as always like you might have done on 22H2. There were some people who had issues on the threads here, but if it worked for you last year it's likely to work for you this year, albeit with a bump in the road above. I'd try with the ISO first on the basis that the toggles are still set. If it moans then unpick. 

Partners machine was fine, mine I'd got those combination of issues above (ISO not opening in the file manager and incompatible of last years toggle script with this year's... latter is what it seemed, not an issue to sort out if you saved last years release of the project, obey run ISO as admin and toggle don't do updates during installation). Another two years of Windows 11 updates on "unsupported" machines thanks to the project. Handy script 👍. Big thanks to the developer.
